app/src/components/adminConsole/csvManager.ts

```diff
@@ -0,0 +1,61 @@
+function snappablesToCsvContent(snappables) {
+    let csvContent = "id,fullName,email,username";
+    snappables.forEach(({ id, fullName, email, username }) => {
+        csvContent += `\n${id},${fullName},${email},${username}`;
+    });
+    return csvContent;
+}
+
+function stringToFileDownload(
+    content: string,
+    filename: string,
+    type = "text/csv;charset=utf-8;"
+) {
+    var blob = new Blob([content], { type });
+    if (navigator.msSaveBlob) {
+        // IE 10+
+        navigator.msSaveBlob(blob, filename);
+    } else {
```

Comment:
This is [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/msSaveBlob) and we don't need to support IE.